node_modules/oae-admin/tenants/tenants.html

```diff
@@ -54,7 +54,11 @@
                                <div class="jeditable-field" data-alias="${tenant.alias|encodeForHTMLAttribute}" data-field="host">${tenant.host|encodeForHTML}</div>
                             </td>
                         {else}
-                            <td><a href="//${tenant.host}" title="${tenant.displayName|encodeForHTMLAttribute}" target="_blank">${tenant.host|encodeForHTML}</a></td>
+                            {if tenant.displayName === tenant.host}
```

Comment:
The tenant displayname is never going to be the same as the tenant host. We can just always have the title attribute here